projects/vassily-kandinsky/index.md

```diff
@@ -0,0 +1,156 @@
+
+![](assets/images/diagonal-lines.png)￼
+
+
+
+In this work he has used two sharp **diagonal lines**. From the bottom left to the top right corner which loses tension according to his rules because the line touches the basic form (edge). Nevertheless, this direction course looks rather harmonious. He selected the colour yellow which inner meaning is rather garish.
+
+The second **diagonal line** runs from the bottom right to the top left corner this loses tension by the touch of the basic form (edge) like the other one. This direction looks unharmonious and he chose a mixture from green and blue, it's a quiet colour. 
+Vassily generates by these both cutting diagonal lines a perspective and shifted the focus point through the crossroads into the circle, this is the gestalt law of space through colour.
+
+![](assets/images/circles-and-lines.png)￼
+
+
+
+Within his work there are 26 overlapping **circles** in different sizes and colours. The gestalt laws are of proximity, of similarity and the connected elements. It has 12 red, darkly violet till orange, circles which appears passive. Moreover, a red circle has a black small circle as the centre and three red circles are bordered. 
+There are also three yellow circles which appears more active and two of them are very big.
+Then there are another six light blue to deep blue circles.
+The deeper the blue is the more it calls the viewer into the infinite and awakes in him the longing for the pure and the extrasensory. 
+A blue tone which sinks more to black gets an overtone of a not human grief, that is the theory of him. 
+There are another three green and a grey circle in the painting. 
+
+Inside of the big circle there run 12 short and 15 long randomly distributed **lines** with different widths and angles with the result that they cut the single circles.
+ 
+Generally Vassily has spoken in his book of warm and cold forms. The vertical ones are the warm forms and the horizontal ones are the cold forms. The internal sound (warmly / coldly) of a diagonal lines determines the angle. 
+The single colours, be it those of the circles or the diagonal lines in the background, are  added through the overlapping, so that the colours are mixed with each other. This is the gestalt law of the space through overlapping.
+
+## Set of rules 
+
+1. Background yellow / white 
+2. two coloured diagonal lines
+3. big black outlined circle, in the middle of the picture 
+
+within the circle 
+
+4. 26 circles = 12 red (1 with inner circle), 6 blue, 3 yellow, 3 greens, 1 grey  
+5. overlapping = addition of both colours 
+6. 12 short and 15 long randomly distributed lines
+
+## Code Result
+
+![](assets/images/colour-kandinsky.png)￼
+###### Source: http://lokeshdhakar.com/projects/color-thief/ 
+
+
+![](assets/images/code.png)￼
+
+Here is the [Link](https://gist.github.com/paulinalinaa/0d57b03f04b118aba5634dedf7360082) of this code.
+
+
+- - -
```

Comment:
This will not create a horizontal line. use  

   
    ---
    
with a new line before and after that.